c10/util/Logging.cpp

```diff
@@ -197,9 +197,7 @@ C10_DEFINE_int(
 // declaring it here. This is a hack but has been used by a bunch of others too
 // (e.g. Torch).
 namespace google {
-namespace glog_internal_namespace_ {
 bool IsGoogleLoggingInitialized();
```

Comment:
Do we still need the decl here? Does glog header provides it already?

Reply:
GLog's `glog/logging.h` header contains declaration of `IsGoogleLoggingInitialized`, so this re-declaration in PyTorch's `c10/util/Logging.cpp` is unnecessary.